WalletWasabi/WabiSabi/Client/CoordinatorClient.cs

```diff
@@ -0,0 +1,100 @@
+using NBitcoin;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using WalletWasabi.Crypto;
+using WalletWasabi.Crypto.ZeroKnowledge;
+using WalletWasabi.Helpers;
+using WalletWasabi.WabiSabi.Backend.PostRequests;
+using WalletWasabi.WabiSabi.Crypto;
+using WalletWasabi.WabiSabi.Models;
+
+namespace WalletWasabi.WabiSabi.Client
+{
+	public class CoordinatorClient
+	{
+		public CoordinatorClient(WabiSabiClient wabiSabiClientAmount, WabiSabiClient wabiSabiClientWeight, IRequestHandler requestHandler)
+		{
+			WabiSabiClientAmount = wabiSabiClientAmount;
+			WabiSabiClientWeight = wabiSabiClientWeight;
+			RequestHandler = requestHandler;
+		}
+
+		public WabiSabiClient WabiSabiClientAmount { get; }
```

Comment:
Rename to WabiSabiClientAmount => AmountCredentials?

Reply:
FWIW in other languages (haskell, Go) I usually like naming things for qualified import conventions, so in general having a `WabiSabi` prefix everywhere is giving me anxiety ;-)

What's the idiomatic way in C#?

Reply:
Yes, I agree. We need to rename this properties but it has to be clear they are _Clients_ and the _WabiSabi_ prefix sucks and adds no meaningful info so I think `AmountCredentialsClient` is the best that I can think about.

Reply:
I've just realized that `WabiSabi` is synonym of `Credentials` then, `AmountCredentialClient` is basically the same. It still sounds too long.

Reply:
i don't think that's too long, and there's no redundancy in `AmountCredentialClient`. i think that's good enough for now